### DISCUSSION TITLE
Background linear gradient customize

### DISCUSSION THREAD

**Discussion by ata58011** [tailwindlabs/tailwindcss#11325] [Help] [answered]
I am trying to customize background with linear gradients colors that i choose but i couldn"t do that can you guys help me with that ?

**Comment by wongjn** [comment, marked as answer]
Use the [`from-*`, `via-*` and/or `to-*` utility classes](https://tailwindcss.com/docs/gradient-color-stops), otherwise, you could use an [arbitrary value class for `background-image`](https://tailwindcss.com/docs/background-image#arbitrary-values).
